nimare/meta/cbma/base.py

```diff
@@ -218,18 +204,31 @@ def _summarystat_to_p(self, stat_values, null_method="analytic"):
         """
         if null_method.startswith("analytic"):
             assert "histogram_bins" in self.null_distributions_.keys()
-            assert "histogram_weights" in self.null_distributions_.keys()
+            assert "histweights_corr-none_method-analytic" in self.null_distributions_.keys()
 
             p_values = nullhist_to_p(
                 stat_values,
-                self.null_distributions_["histogram_weights"],
+                self.null_distributions_["histweights_corr-none_method-analytic"],
                 self.null_distributions_["histogram_bins"],
             )
+
         elif null_method == "empirical":
-            assert "empirical_null" in self.null_distributions_.keys()
+            assert "histogram_bins" in self.null_distributions_.keys()
+            assert "histweights_corr-none_method-empirical" in self.null_distributions_.keys()
+            p_values = nullhist_to_p(
+                stat_values,
+                self.null_distributions_["histweights_corr-none_method-empirical"],
+                self.null_distributions_["histogram_bins"],
+            )
+
+        elif null_method == "reduced_empirical":
+            assert "values_corr-none_method-reducedEmpirical" in self.null_distributions_.keys()
             p_values = null_to_p(
-                stat_values, self.null_distributions_["empirical_null"], tail="upper"
+                stat_values,
+                self.null_distributions_["values_corr-none_method-reducedEmpirical"],
+                tail="upper",
             )
+
         else:
             raise ValueError("Argument 'null_method' must be one of: 'analytic', 'empirical'.")
```

Comment:
should we add `reduced_empirical`?
```suggestion
            raise ValueError("Argument 'null_method' must be one of: 'analytic', 'empirical', 'reduced_empirical'.")
```

Reply:
Not sure. I think @tyarkoni wanted to leave it undocumented, since the reduced empirical method should only be used for testing once we have the full empirical method merged in.